api/src/main/java/org/openmrs/Concept.java

```diff
@@ -861,13 +858,7 @@ public ConceptName getShortNameInLocale(Locale locale) {
 				if (nameLocale.equals(locale)) {
 					return shortName;
 				}
-				// test for partially locale match - any language matches takes precedence over country matches.
```

Comment:
I have deleted this block, because it was redundant. The same code was called whether "if" was true or false